src/main/java/com/bina/varsim/fastqLiftover/MafRecord.java

```diff
@@ -17,7 +18,7 @@ public MafEntry(String entry) {
             if (fields.length != 7) throw new RuntimeException("Unexpected MAF line: "+entry);
             if (! fields[0].equals("s")) throw new RuntimeException("Unexpected MAF line: "+entry);
             src = fields[1];
-            start0 = Integer.parseInt(fields[2]);
+            start = new Position0(Long.parseLong(fields[2]));
```

Comment:
You may want to add a parseSomething static method to Position to make life easier.